doc/chapter-cmdline.asciidoc

```diff
@@ -8,9 +8,10 @@ the screen, similar to tools like vi(m) or mutt. You can now enter commands.
 Pressing the "Enter" key executes the command (possibly giving feedback to the
 user) and closes the commandline. You can cancel entering commands by pressing
 the "Esc" key. The history of all the commands that you enter will be saved to
-_~/.newsboat/history.cmdline_. The backlog is limited to 100 entries by default,
-but can be influenced by setting the <<history-limit,`history-limit`>> configuration variable.
-To disable history saving, set the `history-limit` to `0`.
+the _history.cmdline_ file, stored next to the cache file. The backlog is
```

Comment:
Just simply "cache file"? The more precise "_cache.db_ file" could when searching for the file.